lib/internal/sync/controller/note_controller.dart

```diff
@@ -12,44 +12,40 @@ import 'package:provider/provider.dart';
 import 'package:http/http.dart' as http;
```

Comment:
why did this file get unformatted this commit

Reply:
Accidentally reformatted all files, reverting it probably changed it. I'll format it again and commit it. Honestly you should really squash all these commits into one. I am not proud of this mess of a tree.

Reply:
latest commit should fix it

Reply:
will squash no probs